tensorboard/uploader/uploader_subcommand.py

```diff
@@ -399,113 +421,19 @@ def _die_if_bad_experiment_description(description):
 class UploadIntent(_Intent):
     """The user intends to upload an experiment from the given logdir."""
```

Comment:
Consider adding a small comment clarifying that TB.dev is being shutdown, and this is no longer a supported operation.

Reply:
Done.